expense-reports.fsproj

```diff
@@ -34,6 +34,7 @@
 
   <ItemGroup>
     <Compile Include="Models\*.fs" />
+    <Compile Include="Handlers\*.fs" />
```

Comment:
Did you add this manually or was there some VSCode or other tool magic that was able to do it for you?

Reply:
Manually